app/src/main/kotlin/maru/app/MaruApp.kt

```diff
@@ -207,7 +210,8 @@ class MaruApp(
           finalizationStateProvider = finalizationProvider,
         )
       }
-
+    val forkTransitionSubscriptionManager = InOrderFanoutSubscriptionManager<ForkSpec>()
```

Comment:
I still don't understand why do we need `InOrderFanoutSubscriptionManager` instance here